File: Sources/gestion-culturelle-back/src/main/java/com/afpa/cda/dto/ArticleDto.java
package com.afpa.cda.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Builder
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@ToString
public class ArticleDto {

	private int id;
	
	private ManifestationDto manifestation;
	private PanierDto panier;
	
	int quantite;
}

File: Sources/gestion-culturelle-front/project/src/app/model/articleDto.ts
import { ManifestationDto } from './manifestationDto';
import { PanierDto } from './panierDto';

export class ArticleDto {
    id: number;
    manifestation: ManifestationDto;
    panier: PanierDto;
    quantite: number;


    constructor(id?: number,
        manifestation?: ManifestationDto,
        panier?: PanierDto,
        quantite?: number) {
        this.id = id;
        this.manifestation = manifestation;
        this.panier = panier;
        this.quantite = quantite

    }
}


